src/app/hooks/useClickTracker/index.test.jsx

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable react/prop-types */
```

Comment:
I wrote some tests around error handling to ensure the. view tracker hook doesn't break that app unexpected data is passed to it https://github.com/bbc/simorgh/pull/9071/files#diff-81136bbf60a2de9e097e285911100a0850d8b5dc7985bee4376851cf43e47da8R187

You could literally copy/paste these and change the tests to use your hook and then make those tests pass by changing your hook code.

Reply:
Awesome, thanks, I'll do that now 👍